lib/schemas/global/apiDefendantCourtApplicationSummary.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",

```

Comment:
This is a pain, I know, but to keep swagger up to date you're going to need to update swagger/v2/defendant_summary.json to indicate that the defendant summary is now going to include an array of application_summaries.